OneCommunityGlobal/HGNRest
# Sriram taking over TimeLogger for BMdashboard

# Description
Implementing "Time Logger" Function for bmdashboard.
http://localhost:3000/bmdashboard/timelog

## Related PRS (if any):
This PR should be tested along with [Frontend 2494](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2494)

## Main changes explained:
Please see [Frontend 2494
](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2494)

## How to test:
1. check into current branch
2. do `npm install` and `...` to run `npm run dev` locally
3. Clear site data/cache
4. log as any user
5. Please see [Frontend 2494
](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2494)

## Screenshots or videos of changes:
Please see [Frontend 2494
](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2494)


## Note:
Please see [Frontend 2494
](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2494)
